samples/stock/xaxis/overscroll-pixel/demo.js

```diff
@@ -0,0 +1,39 @@
+Highcharts.setOptions({
+    global: {
+        useUTC: false
+    }
+});
+
+// Create the chart
+Highcharts.stockChart('container', {
+    title: {
+        text: 'Overscroll set in pixels'
+    },
+
+    xAxis: {
+        overscroll: '600px'
```

Comment:
Q: What do you think about a lower value? Like 50px? I think it's a more realistic value (a value to reserve some space for eg labels, annotations, predictions/forecastings)

Reply:
When I added a big pixel value, I thought that it could show the user that the overscroll is limited to 90% of the axis width at some point, but you're right - 50px is a more realistic value for an overscroll.